msf4j/msf4jspring/mvnw.cmd

```diff
@@ -0,0 +1,143 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
We don't need the maven wrappers at this level. What we have at the parent level is fine, so let's remove these.